.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Remove `.vscode` folder from the PR please.

Reply:
- [x] done